sample/src/main/java/io/github/boguszpawlowski/composecalendar/sample/CustomSelectionSample.kt

```diff
@@ -1,22 +1,42 @@
 package io.github.boguszpawlowski.composecalendar.sample
 
+import androidx.compose.foundation.gestures.rememberScrollableState
+import androidx.compose.foundation.gestures.scrollable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.getValue
 import androidx.compose.runtime.mutableStateOf
 import androidx.compose.runtime.remember
 import androidx.compose.runtime.saveable.Saver
 import androidx.compose.runtime.saveable.rememberSaveable
 import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
 import io.github.boguszpawlowski.composecalendar.Calendar
 import io.github.boguszpawlowski.composecalendar.CalendarState
-import io.github.boguszpawlowski.composecalendar.header.MonthState
+import io.github.boguszpawlowski.composecalendar.states.ModeState
 import io.github.boguszpawlowski.composecalendar.selection.SelectionState
+import io.github.boguszpawlowski.composecalendar.states.CurrentState
+import io.github.boguszpawlowski.composecalendar.states.DayEvent
+import io.github.boguszpawlowski.composecalendar.states.EventState
 import java.time.LocalDate
 import java.time.YearMonth
 
 @Composable
 fun CustomSelectionSample() {
-  Calendar(calendarState = rememberMonthSelectionState())
+  val scrollState = rememberScrollState()
+  Column(modifier = Modifier
+    .fillMaxWidth()
+    .verticalScroll(scrollState)
+  ) {
+    val calendarState = rememberMonthSelectionState(
+      eventState = EventState( dayEventList )
+    )
+    ModeControls(modeState = calendarState.modeState)
```

Comment:
I don't think adding mode controls to every sample is necessary

Reply:
Fine.